content/master/concepts/metrics.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Metrics
+weight: 60
+description: "Metrics are essential for monitoring Crossplane's operations, helping to quickly identify and resolve potential issues."
+---
+
+This page offers explanations of various metrics gathered from Crossplane, which are essential for effective monitoring and alerting in your Crossplane environment.
+Understanding these metrics help you maintain the health and performance of your resources, ensuring to identify and address any issues.
+Please note that this document focuses on Crossplane specific metrics and doesn't cover standard Go metrics.
```

Comment:
It'd be useful to mention:

* That these are prometheus style metrics
* Where they're exposed (i.e. the port and URL)
* How to enable them (iirc they're not on by default)

I don't think it's worth explaining what Prometheus style metrics are or how they work, but we should link to the Prometheus documentation for those who are unfamiliar.